### PR TITLE
Feature/item document service

### PR DESCRIPTION
**Pull request by mtrupkin**

### CODE REVIEW

File: scoring/src/main/java/tds/score/services/impl/ContentServiceImpl.java [hunk 1]
```diff
@@ -116,17 +115,16 @@ else if (rubricContentSource == RubricContentSource.ItemXML) {
...
-        try {
-            URI uri = new URI(xmlFilePath);
-            IITSDocument document = itemDocumentService.loadItemDocument(uri, accommodations.getValue(), true);
-            return document;
-        } catch (URISyntaxException e) {
-            throw new ReturnStatusException(e);
-        }
+    @Override
+    @Cacheable(CacheType.LONG_TERM)
+    public IITSDocument getContent(final String itemPath, final AccLookup accommodations) throws ReturnStatusException {
```
Comment (millerts) [inline scoring/src/main/java/tds/score/services/impl/ContentServiceImpl.java]: I think there should be a corresponding unit test change for this class.

Comment (millerts) [inline scoring/src/main/java/tds/score/services/impl/ContentServiceImpl.java]: I realize there may not be an existing test because we were cutting corners as we got close to the release, myself included, but we're post release so I think we should go back to writing tests.

File: scoring/src/main/java/tds/score/configuration/SupportApplicationConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ *
+ * Unless required under applicable law or agreed to in writing, software distributed under the
+ * License is distributed in an “AS IS” BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for specific language governing permissions
+ * and limitations under the license.
+ **************************************************************************************************/
+
+package tds.score.configuration;
+
+public class SupportApplicationConfiguration {
```
Comment (millerts) [inline scoring/src/main/java/tds/score/configuration/SupportApplicationConfiguration.java]: Why is this needed?  The defined constant is only used in the service impl?

File: scoring/src/main/java/tds/score/configuration/ScoringProperties.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+package tds.score.configuration;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "exam-service")
+public class ScoringProperties {
+    private String contentUrl;
+
+    public String getContentUrl() {
```
Comment (millerts) [inline scoring/src/main/java/tds/score/configuration/ScoringProperties.java]: Can you add a JavaDoc to this getter?

File: scoring/src/main/java/tds/score/configuration/ScoringConfiguration.java [hunk 4]
```diff
@@ -33,6 +36,7 @@
 import tds.score.services.impl.ItemScoringServiceImpl;
 
 @Configuration
+@EnableConfigurationProperties(ScoringProperties.class)
```
Comment (millerts) [inline scoring/src/main/java/tds/score/configuration/ScoringConfiguration.java]: Does Spring not inject the properties class directly once you add it to the getContentService `@Bean` annotated method?

File: scoring/src/main/java/tds/score/services/impl/ContentServiceImpl.java (hunk 1 above)
Comment (mtrupkin) [inline scoring/src/main/java/tds/score/services/impl/ContentServiceImpl.java]: Agreed.  Was working on a unit test and got hung up.  I wanted to get the code reviewed while I get past the issues.

**Comment by mtrupkin** [comment]
I see what you want with the ContentService mimicking the RubricService.  I can add that.
